### PR TITLE
Log errors for both regular and subscription errors. and sentry.

### PR DESCRIPTION
**Pull request by hedvigoscar**
Sometimes you make me a little disappointed, apollo.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -13,8 +13,33 @@ import { loggingMiddleware } from './middlewares/logger'
 import { makeSchema } from './schema'
 import { factory } from './utils/log'
 
+import * as Sentry from '@sentry/node'
+Sentry.init({
+  dsn: config.SENTRY_DSN,
+  enabled: Boolean(config.SENTRY_DSN),
+  environment: process.env.NODE_ENV,
```
Comment (palmfjord) [inline src/index.ts]: imo this should be a separate env var. we probably want different environments for `staging` and `production` for example, but `NODE_ENV` _should_ be `production` for both

File: src/index.ts [hunk 2]
```diff
@@ -13,8 +13,33 @@ import { loggingMiddleware } from './middlewares/logger'
...
+})
+
 const logger = factory.getLogger('index')
 
+const handleError = (error: GraphQLError): void => {
+  logger.error(
+    'Uncaught error in GraphQL. Original error: ',
+    error.originalError,
+  )
+  Sentry.captureException(error)
```
Comment (palmfjord) [inline src/index.ts]: would it be possible to do some more lovin with scoping? I think this'll kinda catch all errors and breadcrumbs from the beginning of time

Comment (hedvigoscar) [reply src/index.ts]: I'm not sure that I follow 🙁 I'd love to fix this however!

Comment (palmfjord) [reply src/index.ts]: I'm thinking something like this https://github.com/HedvigInsurance/web-onboarding/blob/master/src/server/middleware/enhancers.ts#L88-L100

Comment (hedvigoscar) [reply src/index.ts]: From what I can tell Apollo Server eats the error before it comes back to Koa, making a regular Koa Middleware not the ideal approach (?). Nor does Apollo Server offer any nice places to perform the 'wrapping' we're looking for in an around-style middleware.

Apollo is starting to make me a little bit sad 🙁

Comment (hedvigoscar) [reply src/index.ts]: New implementation - Ping! @palmfjord

File: src/schema.ts [hunk 3]
```diff
@@ -49,7 +51,8 @@ const makeSchema = async () => {
     schemas: [transformedTranslationSchema, localSchema],
   })
 
-  return schema
+  // return schema
```
Comment (palmfjord) [inline src/schema.ts]: 🙈

Comment (hedvigoscar) [reply src/schema.ts]: 🙈 consider using the suggestions feature for this! Perfect use case I think 🙂

**Pull request merged**